zap/src/main/java/org/parosproxy/paros/view/View.java

```diff
@@ -95,6 +95,7 @@
 // ZAP: 2022/09/21 Use format specifiers instead of concatenation when logging.
 // ZAP: 2022/09/27 Added dialog methods with ZapHtmlLabel parameter.
 // ZAP: 2023/01/10 Tidy up logger.
+// ZAP: 2024/08/20 Warp string dialog messages in ZapLabel so that they can be copied.
```

Comment:
OOps saw this just as I hit approve.

wrap not warp 🤣

Reply:
Ha! Fixed 😁